src/components/Reports/ReportsFilters/ReportsFilters.js

```diff
@@ -4,6 +4,8 @@ import * as S from './ReportsFilters.styles';
 import { FormattedMessageMarkdown } from '../../../i18n/FormattedMessageMarkdown';
 import HeaderSection from '../../shared/HeaderSection/HeaderSection';
 
+export const placeholderDate = '--/--/----';
```

Comment:
It seems to be a different change, maybe a different behavior commit?

Reply:
it can go in the same commit of the use of spinners, because it is a placeholder text that is used while looking for the last visit